dev-libs/libdbh/libdbh-5.0.22.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
```

Comment:
if you want to maintain it please update the code properly, EAPI=5 is obsolete, therefore ebuild has some obsolete parts as well.

Reply:
Thanks for the feedback Zlogene.
Actually I did not think it was mandatory to switch to a newer EAPI since it's mentioned on the EAPI wiki page that "EAPIs 0 through 6 are defined and officially supported".
I will commit the necessary changes to update to the latest EAPI 7 then.

Reply:
well, right, the key word is __supported__, but it does npt actually mean you should use obsole thingy,, morover, EAPI=5 is deprecated and it is not allowed ro commit any new ebuilds with it

Reply:
Ok, thanks for the clarification.

It would then be beneficial to update the eapi-specific page of the devmanual (https://devmanual.gentoo.org/ebuild-writing/eapi/), since it still mentions the following:

"When writing new ebuilds developers can choose whathever EAPI they think is the best. Using the features of the latest EAPI is encouraged."